content/desktop/release-notes.md

```diff
@@ -21,6 +21,56 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1) to see what's coming next.
 
+## 4.34.0
+
+{{< release-date date="2024-08-29" >}}
+
+{{< desktop-install-v2 all=true beta_win_arm=true version="4.34.0" build_path="/165256/" >}}
+
+### New
+
+- [Host networking](/engine/network/drivers/host.md#docker-desktop) support on Docker Desktop is now generally available.
+- If you authenticate via the CLI without Docker Desktop installed, you can now authenticate through a browser-based flow, removing the need for manual PAT generation.
+- Windows now supports automatic reclamation of disk space in Docker Desktop for WSL2.
+- Windows registry key sign in enforcement method in GA.
+- Fresh installations of Docker Desktop now use the containerd image store by default.
+- Improved the output of `docker image list` to show multi-platform-related image information.
+- [Compose Bridge](/compose/compose-bridge/_index.md) (Experimental) is now available from the Compose file viewer. Easily convert and deploy your Compose project to a Kubernetes cluster.
```

Comment:
@aevesdocker is this the correct path? I was somehow not able to find it locally 🙈

Reply:
thanks!!